src/picker/styles/global.scss

```
@@ -12,6 +12,7 @@
   flex-direction: column;
   background: var(--background);
   border: var(--border-size) solid var(--border-color);
+  border-radius: var(--picker-border-radius);
```

```suggestion
  border-radius: var(--border-radius);
```

This maps better with the existing `--border-size`, plus we don't need to specify `picker` because you have to target the picker itself to override this.